experiments/ClimaEarth/setup_run.jl

```diff
@@ -327,7 +327,9 @@ function CoupledSimulation(config_dict::AbstractDict)
         ocean_fraction = FT(1) .- ice_fraction .- land_fraction
 
         if sim_mode <: CMIPMode
-            ocean_sim = OceananigansSimulation(ocean_fraction; output_dir = ocean_output_dir, comms_ctx)
+            stop_date = date(tspan[end])
```

Comment:
`tspan[end] == t_end` is usually a period of time (e.g. `10days`). I believe taking the date like this assumes `t_start == 0`, and we instead want to do something like `start_date + tspan[end]`

Reply:
Ah yeah, thank you!